modules/react/src/components/export-video/export-video-panel-preview.tsx

```diff
@@ -4,10 +4,10 @@
 /* global window */
 
 import React, {Component, RefObject} from 'react';
-import DeckGL, {DeckGLRef} from '@deck.gl/react/typed';
+import DeckGL, {DeckGLRef} from '@deck.gl/react';
 import {MapRef, StaticMap, StaticMapProps} from 'react-map-gl';
-import {MapboxLayer} from '@deck.gl/mapbox/typed';
-import type {DeckProps, MapViewState} from '@deck.gl/core/typed';
+import {MapboxOverlay as MapboxLayer} from '@deck.gl/mapbox';
```

Comment:
Oh, this actually isn't a drop-in replacement and will require some changes to work. http://deck.gl/docs/upgrade-guide#deckglmapbox

Reply:
As a temporary fix I believe we can still import MapboxLayer with `import MapboxLayer from '@deck.gl/mapbox/mapbox-layer'`

Reply:
Rather get this out of the way now. Im not getting any type errors, and all instances in the `src/` folder implement the same props `{id: ..., ...deck}` where id is a string and deck is `DeckProps`

Reply:
@chrisgervang

I replaced the deck instance with the deck props in 03a2f0f1f3a1b50bbfb747f2b558ad6e6376065a

I see that the link shows that MapboxLayer to MapboxOverlay shows `interleaved: true` as the replacement. Should the updated API include this?